torch/_inductor/codecache.py

```diff
@@ -392,6 +558,7 @@ class CompiledFxGraph:
     device_idxs: Set[int] = field(default_factory=set)
     mutated_inputs: Set[str] = field(default_factory=set)
     mutated_input_idxs: Set[int] = field(default_factory=set)
+    constants: Dict[str, torch.Tensor] = field(default_factory=dict)
```

Comment:
I discovered we need to keep these constants so that we can add them as module attributes. That's what happens on the "non-cache" code path and I found 2 torchbench benchmarks that were failing with: `CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.`